config/config.go

```diff
@@ -19,7 +19,7 @@ type Config struct {
 	Account    account.Config
 	Validators []node.ValidatorConfig
 	Builders   []node.BuilderConfig
-	FullNode   node.FullNodeConfig
+	CommonNode node.ChainConfig
```

Comment:
In Ethereum, it is usually called `Web3Providers`, I think `RPCProviders` is ok here.

Reply:
done